### PR TITLE
Initialize Rust-backend using Axum

### LINKED ISSUE #1: Init backend

### PR DESCRIPTION
**Pull request by LarsSelbekk**
Closes #1

### CODE REVIEW

**Review by mathiazom — CHANGES_REQUESTED** [review]
I think it makes sense to maintain a seperate Rust-specific .gitignore for the backend directory, in addition to the top level one.

File: backend/Cargo.toml [hunk 1]
```diff
@@ -6,3 +6,4 @@ edition = "2021"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
+axum = { version = "0.6.12", features = ["http2"] }
```
Comment (redrei) [inline backend/Cargo.toml:9]: hvorfor 0.6.12 og ikke den nyeste releasen [0.7.2](https://github.com/tokio-rs/axum/releases/tag/axum-extra-v0.7.2)?

**Review by redrei — APPROVED** [review]
Okay init

**Review by ulrik2204 — APPROVED** [review]
LGTM

**Review by mathiazom — APPROVED** [review]

**Pull request merged**